drivers/apcupsd-ups.h

```diff
@@ -133,5 +133,10 @@ static apcuspd_info_t nut_data[] =
 	{ "MINTIMEL", "battery.runtime.low", ST_FLAG_RW, 60, "%.0f", DU_FLAG_NONE, NULL },
 	{ "RETPCT", "battery.charge.restart", ST_FLAG_RW, 1, "%1.1f", DU_FLAG_NONE, NULL },
 	{ "NOMPOWER", "ups.realpower.nominal", 0, 1, "%1.1f", DU_FLAG_INIT, NULL },
+	{ "LOAD_W", "ups.realpower", 0, 1, "%1.1f", DU_FLAG_INIT, NULL },
```

Comment:
I think some of the flags are switched around. `ups.realpower` should be updated regularly, so it should have `DU_FLAG_NONE` (same for `ups.power`). `ups.power.nominal` won't change.